src/RestEase/Implementation/Requester.cs

```diff
@@ -404,6 +406,14 @@ protected virtual async Task<HttpResponseMessage> SendRequestAsync(IRequestInfo
             return response;
         }
 
+        private void ApplyHttpRequestMessageProperties(IRequestInfo requestInfo, IDictionary<string, object> messageProperties)
```

Comment:
Nit: Can we make this `protected virtual` and document it?

Reply:
Here I thought that the method does entirely single purpose thing and there is probably nothing meaningful to customize but sure.

Reply:
I guess that's fair enough, let's keep it as-is.